README.md

```diff
@@ -288,9 +291,29 @@ make test_unit
 make test_e2e_shannon_relay
 ```
 
-## 8. Troubleshooting
+## 8. Running Localnet
 
-### 8.1. Docker Permissions Issues - Need to run sudo?
+You can use path configuration under `/local` to spin up a local development environment using `Kind` + `Tilt`.
+
+### 8.1 Pre-requisites
+
+Make sure you have the following tools installed:
+
+- [Kind](https://kind.sigs.k8s.io/#installation-and-usage)
```

Comment:
We have a pre-requisites section in `1.1`. Can you move it there?

Reply:
Moved the pre-requisites to deployment section.